docs/product-getting-started-guide/src/main/asciidoc/task_business-rules.adoc

```diff
@@ -1,108 +1,177 @@
 [id='_defining_business_rules']
-= Defining Business Rules
+= Defining the Driver License Suspension Rule
```

Comment:
Similar feedback as with chap. 2. Needs to follow the templates. I'd restructure this as follows (if it were me):

**3. Creating traffic violation and driver suspension rules** <Concept module>
<Context statement - Why these two rules are important and where they will be created (GDT, GRE)>
**3.1. Creating a driver license suspension rule** <Procedure module>
<Context statement - What this rule does and why important, probably repeating a portion of the concept>
**.Procedure**
<Steps>
**3.1.1. Setting the suspension rule conditions** <Procedure module>
<Context statement - What conditions are, the role they play in a rule.>
**.Procedure**
<Steps>
**3.1.2. Setting the suspension rule actions** <Procedure module>
<Context statement - What actions are, the role they play in a rule.>
**.Procedure**
<Steps>

Again, each segment a separate con or proc module/file.

Reply:
I made some initial layout changes. I will revisit after I complete the steps for the last chapters. Need to move forward on that for now. You can take a look at my latest changes at http://file.rdu.redhat.com/~mhaglund/BXMSDOC-2032/

Time may not be on our side for a perfect modularized doc. But, I will do my best to follow the guidelines.